On prefix, we want the versions from within EPREFIX, so just search the PATH -- don't
try to be clever about where the tools may be found

--- acl-2.2.47.orig/m4/package_utilies.m4
+++ acl-2.2.47/m4/package_utilies.m4
@@ -22,57 +22,39 @@ AC_DEFUN([AC_PACKAGE_UTILITIES],
     AC_SUBST(cc)
     AC_PACKAGE_NEED_UTILITY($1, "$cc", cc, [C compiler])
 
-    if test -z "$MAKE"; then
-        AC_PATH_PROG(MAKE, gmake,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
-    fi
-    if test -z "$MAKE"; then
-        AC_PATH_PROG(MAKE, make,, /usr/bin)
-    fi
+    AC_PATH_PROG(MAKE, gmake)
+    AC_PATH_PROG(MAKE, make)
     make=$MAKE
     AC_SUBST(make)
     AC_PACKAGE_NEED_UTILITY($1, "$make", make, [GNU make])
 
     AC_PROG_LIBTOOL
 
-    if test -z "$TAR"; then
-        AC_PATH_PROG(TAR, tar,, /usr/freeware/bin:/bin:/usr/local/bin:/usr/bin)
-    fi
+    AC_PATH_PROG(TAR, tar)
     tar=$TAR
     AC_SUBST(tar)
-    if test -z "$ZIP"; then
-        AC_PATH_PROG(ZIP, gzip,, /bin:/usr/bin:/usr/local/bin:/usr/freeware/bin)
-    fi
+    AC_PATH_PROG(ZIP, gzip)
 
     zip=$ZIP
     AC_SUBST(zip)
 
-    if test -z "$MAKEDEPEND"; then
-        AC_PATH_PROG(MAKEDEPEND, makedepend, /bin/true)
-    fi
+    AC_PATH_PROG(MAKEDEPEND, makedepend, /bin/true)
     makedepend=$MAKEDEPEND
     AC_SUBST(makedepend)
 
-    if test -z "$AWK"; then
-        AC_PATH_PROG(AWK, awk,, /bin:/usr/bin)
-    fi
+    AC_PROG_AWK
     awk=$AWK
     AC_SUBST(awk)
 
-    if test -z "$SED"; then
-        AC_PATH_PROG(SED, sed,, /bin:/usr/bin)
-    fi
+    AC_PROG_SED
     sed=$SED
     AC_SUBST(sed)
 
-    if test -z "$ECHO"; then
-        AC_PATH_PROG(ECHO, echo,, /bin:/usr/bin)
-    fi
+    AC_PATH_PROG(ECHO, echo)
     echo=$ECHO
     AC_SUBST(echo)
 
-    if test -z "$SORT"; then
-        AC_PATH_PROG(SORT, sort,, /bin:/usr/bin)
-    fi
+    AC_PATH_PROG(SORT, sort)
     sort=$SORT
     AC_SUBST(sort)
 
@@ -80,31 +62,23 @@ AC_DEFUN([AC_PACKAGE_UTILITIES],
     AC_PROG_LN_S
 
     if test "$enable_gettext" = yes; then
-        if test -z "$MSGFMT"; then
-                AC_PATH_PROG(MSGFMT, msgfmt,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
-        fi
+        AC_PATH_PROG(MSGFMT, msgfmt)
         msgfmt=$MSGFMT
         AC_SUBST(msgfmt)
         AC_PACKAGE_NEED_UTILITY($1, "$msgfmt", msgfmt, gettext)
 
-        if test -z "$MSGMERGE"; then
-                AC_PATH_PROG(MSGMERGE, msgmerge,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
-        fi
+        AC_PATH_PROG(MSGMERGE, msgmerge)
         msgmerge=$MSGMERGE
         AC_SUBST(msgmerge)
         AC_PACKAGE_NEED_UTILITY($1, "$msgmerge", msgmerge, gettext)
 
-        if test -z "$XGETTEXT"; then
-                AC_PATH_PROG(XGETTEXT, xgettext,, /usr/bin:/usr/local/bin:/usr/freeware/bin)
-        fi
+        AC_PATH_PROG(XGETTEXT, xgettext)
         xgettext=$XGETTEXT
         AC_SUBST(xgettext)
         AC_PACKAGE_NEED_UTILITY($1, "$xgettext", xgettext, gettext)
     fi
 
-    if test -z "$RPM"; then
-        AC_PATH_PROG(RPM, rpm,, /bin:/usr/bin:/usr/freeware/bin)
-    fi
+    AC_PATH_PROG(RPM, rpm)
     rpm=$RPM
     AC_SUBST(rpm)
 
